gammapy/analysis/core.py

```diff
@@ -121,16 +122,22 @@ def get_observations(self):
         for obs in self.observations:
             log.debug(obs)
 
-    def get_datasets(self):
-        """Produce reduced datasets."""
+    def get_datasets(self, show_pbar=False):
```

Comment:
Maybe rename consistently to the more explicit `show_progress_bar`? I think it's not too long but much more explicit...